src/ocaml/typing/untypeast.ml

```diff
@@ -679,7 +681,9 @@ let module_expr (sub : mapper) mexpr =
           | Tmod_unpack (exp, _pack) ->
               Pmod_unpack (sub.expr sub exp)
               (* TODO , sub.package_type sub pack) *)
-          | Tmod_hole -> Pmod_hole
+          | Tmod_hole ->
+              let id = Location.mkloc "merlin.hole" loc in
+              Pmod_extension (id, PStr [])
```

Comment:
We've imported `Ast_helpers`, so this can just be `Mod.hole ()`, I believe

Reply:
Same.